agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/configuration/ConfigurationBuilder.java

```diff
@@ -301,6 +301,8 @@ private static void overlayConfiguration(
     // only fall back to default sampling configuration after all overlays have been performed
     if (config.sampling.requestsPerSecond == null && config.sampling.percentage == null) {
       config.sampling.requestsPerSecond = 5.0;
+      configurationLogger.info(
+          "Some telemetry may be sampled out because a default sampling configuration was added in version 3.4.0 to reduce the default billing cost. You can set the sampling configuration explicitly: https://learn.microsoft.com/azure/azure-monitor/app/java-standalone-config#sampling");
```

Comment:
we might want to update this message based on https://github.com/microsoft/ApplicationInsights-Java/pull/3713. 
from 3.5.3, customer can disable ingestion sampling?

Reply:
this message is only emitted when customer has no sampling configuration (and so the default sampling configuration is used)